Report/Chapters/Introduction.tex

```diff
@@ -2,7 +2,6 @@ \chapter{Introduction}
 
 Programming has always been an activity which is done on a computer with a keyboard. Despite the existence of different visual programming languages, like Blender, there are no programs in which well known programming languages like Java, Haskell or C(++) can be visually implemented. 
```

Comment:
Is Blender a visual programming language?
Do you perhaps mean the visual programming language used within Blender?